screens/ProfileScreen/components/ProfilePicturePickerModal/ProfilePicturePickerModal.js

```diff
@@ -15,15 +26,15 @@ const ProfilePicturePickerModal = ({
 }) => {
   // selectedImage is an object with properties "fileName", "fileSize", height, isVertical, "originalRotation", width, path, type
   const [selectedImage, setSelectedImage] = useState(null);
-  const [savingLoader, setSavingLoader] = useState(false)
+  const [savingLoader, setSavingLoader] = useState(false);
 
   const pickImageHandler = () => {
     const options = {
       title: 'Pick an Image',
       noData: true,
     };
 
-    ImagePicker.showImagePicker(options, (response) => {
+    ImagePicker.launchImageLibrary(options, (response) => {
```

Comment:
when you merge, the master will have showImagePicker instead of launch camera. showImagePicker is the correct one which allows you to both take a pic and go to galery

Reply:
i agree with Pokumars